src/main/java/org/opensearch/plugin/insights/rules/action/top_queries/TopQueriesRequest.java

```diff
@@ -34,10 +34,10 @@ public class TopQueriesRequest extends BaseNodesRequest<TopQueriesRequest> {
     public TopQueriesRequest(final StreamInput in) throws IOException {
         super(in);
         this.metricType = MetricType.readFromStream(in);
-        this.from = null;
-        this.to = null;
-        this.verbose = null;
-        this.id = null;
+        this.from = in.readOptionalString();
+        this.to = in.readOptionalString();
+        this.id = in.readOptionalString();
+        this.verbose = in.readOptionalBoolean();
```

Comment:
I think we need version checks for reading and writing these params. Older version nodes will not send or expect these.